test/checks/color/color-contrast.js

```diff
@@ -723,6 +723,25 @@ describe('color-contrast', function () {
       assert.deepEqual(checkContext._relatedNodes, []);
     });
 
+    it('should report incomplete for options.contrastRatio.normal.minThreshold', function () {
```

Comment:
I'm confused about what these test titles mean / what this test is supposed to prove. Can you clarify?

Reply:
I was just copying the tests we already had (the prior test to each of these) and modifying it to return incomplete to prove that it reports incomplete since the threshold would have prevent that from happening in the code.

Reply:
... sorry, but I don't understand what this test proves. The way this description reads suggests that any time `normal.minThreshold` is used, the result must be `undefined`. That's clearly not what's going on. But then what is?